applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -612,51 +566,41 @@ int le_audio_pause(void)
 	return 0;
 }
 
-int le_audio_send(struct encoded_audio enc_audio)
-{
-	ARG_UNUSED(enc_audio);
-
-	LOG_WRN("Not possible to send audio data from broadcast sink");
-	return -ENXIO;
-}
-
-int le_audio_enable(le_audio_receive_cb recv_cb, le_audio_timestamp_cb timestmp_cb)
+int broadcast_sink_disable(void)
 {
 	int ret;
 
-	ARG_UNUSED(timestmp_cb);
+	if (active_stream.stream != NULL && active_stream.stream->ep != NULL) {
+		if (active_stream.stream->ep->status.state == BT_BAP_EP_STATE_STREAMING) {
+			ret = bt_bap_broadcast_sink_stop(broadcast_sink);
+			if (ret) {
+				LOG_ERR("Failed to stop sink");
+			}
+		}
+	}
 
-	ret = initialize(recv_cb);
+	ret = bis_headset_cleanup();
 	if (ret) {
-		LOG_ERR("Failed to initialize");
+		LOG_ERR("Error cleaning up");
 		return ret;
 	}
 
-	LOG_DBG("LE Audio enabled");
+	LOG_DBG("Broadcast sink disabled");
 
 	return 0;
 }
 
-int le_audio_disable(void)
+int broadcast_sink_enable(le_audio_receive_cb recv_cb)
```

Comment:
The only thing this function does now is call initialize, so I suggest we just move the code from initialize here.